__tests__/app.test.js

```diff
@@ -266,6 +266,41 @@ describe("The Server", () => {
           });
       });
     });
+    describe("POST", () => {
+      it("Status 201, accepts object to input into comments table and returns comment", () => {
+        return request(app)
+          .post("/api/articles/4/comments")
+          .send({
+            username: "icellusedkars",
+            body: `this might be the greatest masterpiece I have ever read. I mean sheep, who would have guessed it`,
+          })
+          .expect(201)
+          .then(({ body: { comment } }) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                article_id: 4,
+                comment_id: 19,
+                votes: 0,
+                created_at: expect.any(String),
+                author: "icellusedkars",
+                body: "this might be the greatest masterpiece I have ever read. I mean sheep, who would have guessed it",
+              })
+            );
+          });
+      });
+      it("Status 404, when requesting to comment on an article that does not currently exist", () => {
+        return request(app)
+          .post("/api/articles/99999999/comments")
+          .send({
+            username: "icellusedkars",
+            body: `this might be the greatest masterpiece I have ever read. I mean sheep, who would have guessed it`,
+          })
+          .expect(404)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("resource not found");
+          });
+      });
+    });
```

Comment:
I don't know this is all the tests that I would like to see here, there are quite a few 400 and 404's that could happen